### DISCUSSION TITLE
Can't apply for Vroid SDK

### DISCUSSION THREAD

**Discussion by ZenRepublic** [pixiv/vroid-sdk-developers#41] [General]
Hey please help me! I am trying to apply for vroid SDK for my multiplayer VR game but I am getting some japanese error saying "apply failed" Is there something I can do? Tried multiple times

**Comment by nyamadan** [comment]
Could not import VRoid SDK unitypackage?

Are there any errors in the UnityEditor console?

**Comment by alexulter** [comment]
Same problem for me 
![image](https://user-images.githubusercontent.com/2931471/181520717-b25da489-0b17-42a8-aff0-286fe676e3e0.png)
Popups "Try again later" error indefinitely.

**Reply by nyamadan** [reply]
Could you tell me about your development environment?
